database/migrations/2016_06_07_153443_create_smart_lives_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSmartLivesTable extends Migration
```

Comment:
@gabimadd do we really need an extra table and a model? What I thought is to have services table for all types of services. And you can distinguish between them by checking the services parent. If parent = Smartlife Service => the service is obviously of type Smart Life.